realwork/WatchApp Extension/InterfaceController.swift

```diff
@@ -0,0 +1,60 @@
+//
+//  InterfaceController.swift
+//  WatchApp Extension
+//
+//  Created by Nathan Ngobale on 2020/04/29.
+//  Copyright © 2020 Nathan Ngobale. All rights reserved.
+//
+
+import WatchKit
+import Foundation
+import WatchConnectivity //**1
+
+class InterfaceController: WKInterfaceController {
+
+    let session = WCSession.default//**3
+    let colorThemes: [UIColor] = [.yellow, .blue,
+                                .magenta, .red,
+                                .purple, .green
+                                
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Lines should not have trailing whitespace.](https://app.codacy.com/manual/NathanDVT/Musico/pullRequest?prid=5444460)